admin/data/fixtures/check-form-4.csv

```diff
@@ -0,0 +1,11 @@
+Factor 1, Factor 2
```

Comment:
Yes.  The a/c states there should not be a header.

Only check-form-1.csv is good input.  The others are all bad.